### PR TITLE
Created Food Website Template

### LINKED ISSUE #101: Add a Food Website Tempelate
Will add a responsive food web page 
@ankitapuri  please assign it to me

### PR DESCRIPTION
**Pull request by sidparashar2001**
# Description

Created a completely responsive and attractive food website template

Fixes #101

### CODE REVIEW

**Comment by sidparashar2001** [comment]
image

**Comment by arpit456jain** [comment]
@sidparashar2001 Is this from YouTube
Code with Harry ??

**Comment by arpit456jain** [comment]
@sidparashar2001 Is this from YouTube
Code with Harry ??

**Comment by sidparashar2001** [comment]
Yes I took the reference but can check the code and some styling it's bit different

**Comment by arpit456jain** [comment]
Yeah @sidparashar2001 no problem actually same two templates are already present but no worries every contribution counts

**Comment by sidparashar2001** [comment]
Thanks😊

**Comment by arpit456jain** [comment]
https://deploy-preview-103--cool-front-end-templates.netlify.app/food%20website%20template/

I think there is some problem can you please confirm @ankitapuri

File: Food Website Template/index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Best Food delivery services in India|MyFood.In</title>
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" media="screen and (max-width:900px)" href="/css/phone.css">
```
Comment (arpit456jain) [inline Food Website Template/index.html:9]: Just use css/style.css 
Do not use / before it i think because of this it is finding css in root directory

**Review by arpit456jain — CHANGES_REQUESTED** [review]

**Comment by ankitapuri** [comment]
> https://deploy-preview-103--cool-front-end-templates.netlify.app/food%20website%20template/
> 
> I think there is some problem can you please confirm @ankitapuri

yes its only plain html

**Comment by arpit456jain** [comment]
@sidparashar2001 please make required changes asap

**Comment by sidparashar2001** [comment]
Updated the changes Sorry for the mistake

**Comment by sidparashar2001** [comment]
Sorry for the Mistake Updated the changes

**Pull request merged**